beacon-chain/state/v1/BUILD.bazel

```diff
@@ -106,17 +107,17 @@ go_test(
     ],
 )
 
-#ssz_gen_marshal(
-#    name = "ssz_generated_files",
-#    srcs = [
-#        "types.go",
-#    ],
-#    includes = [
-#        "//beacon-chain/state/custom-types:go_default_library",
-#        "//proto/prysm/v1alpha1:go_default_library",
-#        "@com_github_prysmaticlabs_eth2_types//:go_default_library",
-#    ],
-#    objs = [
-#        "BeaconState",
-#    ],
-#)
+ssz_gen_marshal(
+    name = "ssz_generated_files",
+    srcs = [
+        "types.go",
+    ],
+    includes = [
+        "//beacon-chain/state/custom-types:go_default_library",
+        "//proto/prysm/v1alpha1:go_default_library",
+        "@com_github_prysmaticlabs_eth2_types//:go_default_library",
+    ],
+    objs = [
+        "BeaconState[no-htr]",
```

Comment:
The referenced version of fastssz recognizes `[...]` as options. `no-htr` means "do not generate `HashTreeRoot`.